runtime/near-vm-runner/src/tests/fuzzers.rs

```diff
@@ -8,10 +8,12 @@ use near_primitives::version::PROTOCOL_VERSION;
 use near_vm_errors::{FunctionCallError, VMError};
 use near_vm_logic::mocks::mock_external::MockedExternal;
 use near_vm_logic::{VMConfig, VMContext};
+#[cfg(all(feature = "wasmer2_vm", target_arch = "x86_64"))]
 use wasmer_engine::Executable;
 
 use crate::internal::wasmparser::{Export, ExternalKind, Parser, Payload, TypeDef};
 use crate::internal::VMKind;
+#[cfg(all(feature = "wasmer2_vm", target_arch = "x86_64"))]
 use crate::wasmer2_runner::Wasmer2VM;
```

Comment:
maybe move these imports into the function then?

Reply:
Good idea, thanks :)